src/Look/InfoBoxLook.cpp

```diff
@@ -60,8 +58,17 @@ InfoBoxLook::Initialise(bool _inverse, bool use_colors,
 }
 
 void
-InfoBoxLook::ReinitialiseLayout(unsigned width)
+InfoBoxLook::ReinitialiseLayout(const UISettings &settings,
+      unsigned width)
 {
+
+  uint8_t bg = UISettings::GetBgColor(settings);
+  background_color = Color(bg,bg,bg);
+
+  uint8_t fg = inverse ? 255 : 0;
```

Comment:
This whole section of code added by you looks confusing - but what confuses me most is that even in low-contrast mode, it's 100% white.

Reply:
That is because, right now, contrast only changes the background. So foreground is just black or white.